### ISSUE TITLE
Incorrect generation of file conanvcvars.ps1

### ISSUE THREAD

**Issue by vergara-r** [conan-io/conan#16319]
### Describe the bug

The file 'conanvcvars.ps1' is not generated correctly, a backslash is present on the first line of the generated file:

```powershell
\
            if (-not $env:VSCMD_ARG_VCVARS_VER){
                Push-Location "$PSScriptRoot"
                cmd /c "conanvcvars.bat&set" |
                foreach {
                  if ($_ -match "=") {
                    $v = $_.split("=", 2); set-item -force -path "ENV:\$($v[0])"  -value "$($v[1])"
                  }
                }
                Pop-Location
                write-host conanvcvars.ps1: Activated environment}
```

It causes the following error:

```powershell
\ : The term '\' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if a path was included, verify that the 
path is correct and try again.
At C:\Users\vergarar\Documents\GitHub\ipp-api-glib\build\Debug\generators\conanvcvars.ps1:1 char:1
+ \
+ ~
    + CategoryInfo          : ObjectNotFound: (\:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException
```

### How to reproduce it

Use conan version 2.3.1 or 2.3.0 and launch the command in windows powershell:  

<any test project>\build\Debug\generators\conanbuild.ps1

**Comment by AbrilRBS** [comment]
Hi @vergara-r thanks a lot for taking the time to report the issue, we appreciate it

In fact this seems like a typo from https://github.com/conan-io/conan/pull/16000 - we'll fix this for the next release (Either a patch or minor version, I still need to talk with the team) thanks :)

**Comment by memsharded** [comment]
Closed by #16322 for next Conan 2.3.2
